contracts/contracts/BidderRegistry.sol

```diff
@@ -494,6 +519,5 @@ contract BidderRegistry is
         return lockedFunds[bidder][window];
     }
 
-    // solhint-disable-next-line no-empty-blocks
-    function _authorizeUpgrade(address) internal override onlyOwner {}
+    function _authorizeUpgrade(address) internal override onlyOwner {} // solhint-disable no-empty-blocks
```

Comment:
fyi `// solhint-disable` disables for the rest of the file, so that's why `// solhint-disable-next-line no-empty-blocks` existed here previously